Documentation/Contributing/Roadmap.md

```diff
@@ -23,67 +26,106 @@ The Mixed Reality Toolkit will require Unity 2018.4.
 
 > When Unity releases an LTS (Long Term Support) product, the Mixed Reality Toolkit will update to the LTS release. MRTK will also support the latest non-beta (ex: 2019.1) tech branch version of Unity, at the time at which MRTK was released.
 
-### 2.3.0
+### 2.4.0
 
-The version 2.3.0 plan has been finalized. While some details may change as the iteration progresses, the following describes the overall plan for the next MRTK release. For the latest status of the release, please visit the [milestone page]( https://github.com/microsoft/MixedRealityToolkit-Unity/milestone/10).
+For the latest status of the release, please visit the [milestone page]( https://github.com/microsoft/MixedRealityToolkit-Unity/milestone/11).
 
 Status: In development
 
-Timeline: January 2029
+Timeline: March 2020
 
 Themes:
 
 - Stability
 - Developer education
-- Unity 2019.3 XR SDK support
 - User Experience
-- Iterate on mobile AR support
+- Hand Extensibility
```

Comment:
I feel like this is rather small part of it, where a bigger item was actually other platforms (in a maintainable way) - i.e. leap/quest, some of which we go through XR SDK to leverage our new XR SDK work.

Reply:
I can add those as work items

Reply:
i mean themes